src/Examine.Core/Search/IFacetOperations.cs

```diff
@@ -8,36 +8,28 @@ namespace Examine.Search
     /// </summary>
     public interface IFacetOperations : IQueryExecutor
     {
-        /// <summary>
-        /// Add a facet string to the current query
-        /// </summary>
-        /// <param name="field"></param>
-        /// <param name="facetConfiguration"></param>
-        /// <returns></returns>
-        IFacetOperations Facet(string field, Action<IFacetQueryField>? facetConfiguration = null);
-
         /// <summary>
         /// Add a facet string to the current query, filtered by a single value or multiple values
         /// </summary>
         /// <param name="field"></param>
         /// <param name="facetConfiguration"></param>
         /// <param name="values"></param>
         /// <returns></returns>
-        IFacetOperations Facet(string field, Action<IFacetQueryField>? facetConfiguration = null, params string[] values);
+        IFacetOperations FacetString(string field, Action<IFacetQueryField>? facetConfiguration = null, params string[] values);
 
         /// <summary>
         /// Add a range facet to the current query
         /// </summary>
-        IFacetOperations Facet(string field, params DoubleRange[] doubleRanges);
+        IFacetOperations FacetDoubleRange(string field, params DoubleRange[] doubleRanges);
 
         /// <summary>
         /// Add a range facet to the current query
         /// </summary>
-        IFacetOperations Facet(string field, params FloatRange[] floatRanges);
+        IFacetOperations FacetFloatRange(string field, params FloatRange[] floatRanges);
 
         /// <summary>
         /// Add a range facet to the current query
         /// </summary>
-        IFacetOperations Facet(string field, params Int64Range[] longRanges);
+        IFacetOperations FacetLongRange(string field, params Int64Range[] longRanges);
     }
```

Comment:
Is there any specific reason of why the `Facet` naming of the methods has been changed here?

Reply:
Yes. It disambiguates the different Facet methods as the params are optional on all.